usaspending_api/api_docs/api_documentation/federal_account/available_object_classes.md

```diff
@@ -0,0 +1,69 @@
+## Avalible Object Classes - Depreciated
```

Comment:
`Avalible` -> `Available`, and this is not deprecated

Reply:
Done